crt/CMakeLists.txt

```diff
@@ -40,6 +40,12 @@ if(UNIX AND NOT APPLE)
             set(MY_ASSEMBLER_IS_TOO_OLD_FOR_512AVX ON CACHE BOOL "Disable AVX512 on old GCC that not supports it")
         endif()
 
+        string(TOLOWER "${CMAKE_SYSTEM_PROCESSOR}" CMAKE_SYSTEM_PROCESSOR_LOWER)
```

Comment:
added this for https://github.com/aws/aws-lc/commit/6fe8dcbe96e580ea85233fdb98a142e42951b70b to work on manylinux-1-x86, it's not released yet, but we will need this once it's